test/parallel/test-runner-run-watch.mjs

```diff
@@ -12,6 +12,9 @@ import { join } from 'node:path';
 if (common.isIBMi)
   common.skip('IBMi does not support `fs.watch()`');
 
+if (common.isAIX)
```

Comment:
I had a rough time with CI on `AIX` previously, I don't know if we should skip it or just skip the test that has 100% failure rate due to the platform limitation.